### PR TITLE
Doc feedback for the release

### PR DESCRIPTION
**Pull request by sunithasomanna**
Incorporated overall feedback on the Getting Started Readme from the team for the Phase 5 release.

### CODE REVIEW

**Review by chauberahul1993 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -483,7 +483,7 @@ The following table lists the software components and versions that are compatib
     | busybox               | 1.33        | busybox.tar                  |
     | dellplugin            | 2.0         | dellplugin.tar               |
     | lenovoplugin          | 1.0         | lenovoplugin.tar             |
-    | urplugin              | 3.0         | urplugin.tar                 |
+    | urplugin              | 3.1         | urplugin.tar                 |
     | grfplugin             | 3.0         | grfplugin.tar                |
     | telemetry             | 2.0         | telemetry.tar                |
```
Comment (hemkapoor) [inline README.md:490]: Please verify the versions. These are not the updated ones.

File: README.md [hunk 2]
```diff
@@ -120,6 +121,7 @@ Deploying Resource Aggregator for ODIM in a data center involves installing the
     - Tasks
     - Update
     - Telemetry
+    - License
 -   The plugin microservices such as the Dell plugin, URP, and additional third-party plugins
```
Comment (sukhwinder20) [inline README.md:125]: Lenovoplugin is missing

File: README.md (hunk 1 above)
Comment (sunithasomanna) [reply README.md:490]: Adding it on a new PR. Closing this PR because it has issues

File: README.md (hunk 2 above)
Comment (sunithasomanna) [reply README.md:125]: Adding it on a new PR. Closing this PR because it has issues

**Comment by sunithasomanna** [comment]
New PR for these changes - https://github.com/sunithasomanna/ODIM/pull/96

**Pull request closed without merging**